packages/patrol_cli/lib/src/commands/build_android.dart

```diff
@@ -154,6 +154,7 @@ class BuildAndroidCommand extends PatrolCommand {
       packageName: packageName,
       appServerPort: super.appServerPort,
       testServerPort: super.testServerPort,
+      uninstall: false,
```

Comment:
Why is `uninstall` set to `false` for `build` command?

Reply:
I'll go further: why does it exist for the `build` command?